HTML/_JS/jBox.dyalog

```diff
@@ -1,7 +1,7 @@
 ﻿:Class jBox : #._JQ._jqWidget
 ⍝ Description:: jBox widget
 ⍝
-⍝ Constructor:: [type [message [content]]]
+⍝ Constructor:: [Type [Message [Content]]]
```

Comment:
Our convention is to use lower case in constructor arguments and upper case for public Fields